manager/manager.go

```diff
@@ -87,6 +87,11 @@ type Options struct {
 
 	// RenewalBackoffConfig configures the exponential backoff applied to certificate renewal failures.
 	RenewalBackoffConfig *wait.Backoff
+	// IssueRenewalTimeout defines timeout value for each issue() call in renewal process
+	IssueRenewalTimeout time.Duration
+	// IssuePollInterval defines an interval time for each subroutine call (check CSR status) within issue() function
+	// Note, the issuePollInterval should be less than renewalIssueTimeout
```

Comment:
Can you add validation into the `NewManager` function to ensure that `IssuePollInterval` is less than `IssueRenewalTimeout`? (so the error happens immediately instead of getting weird/awful spinning edge-cases)